pynicotine/chatrooms.py

```diff
@@ -385,16 +385,19 @@ def _user_left_room(self, msg):
 
     def update_completions(self):
 
-        self.completion_list = [config.sections["server"]["login"]]
+        if not config.sections["words"]["tab"] and not config.sections["words"]["dropdown"]:
```

Comment:
I don't think these options should be referenced in the core.

How is performance for you since 91331231b2bb14418dbfa6aa16857e6a80ba804b?

Reply:
It's okay, the most expensive operation is always https://github.com/nicotine-plus/nicotine-plus/blob/6736dec3cc75b367f79ac97ecbbda8b1370f3c65/pynicotine/gtkgui/widgets/textentry.py#L195

it seems excessive to clear and compile and rebuild the entire set and model each time a change occurs. I can't help thinking that in the case of a single item changing (such as a user leaving or joining a room) it would be better to get the intersection of the sets (make a copy beforehand) and then just do an add/remove.

Reply:
We could look closer into this at some point. Since completions are now properly unset in master, I'll close the PR.